python-code/set-sample/set-sample1.py

```diff
@@ -1,7 +1,8 @@
 import os
 import math
-import sys
-Value = 1
-Value = 2
-addedValue = 1 + 2
+import sys 
+Value = 1 
+Value = 2 
+addedValue = 1 + 2 
 print(addedValue) 
```

Comment:
This is a trailing whitespace suggestion from GitHubBot
```suggestion 
```print("this is a simple test code")